packages/react-server-webpack-plugin/src/react-server-webpack-plugin.ts

```diff
@@ -76,8 +76,8 @@ export class ReactServerPlugin {
             });
 
           const app = createServer({
-            port: ${port},
-            ip: '${host}',
+            port: ${port || 'process.env.REACT_SERVER_PORT'},
```

Comment:
I suppose we could actually do

```tsx
${port || process.env.NODE_ENV === 'production' ? 'process.env.REACT_SERVER_PORT' : '8080'} 
```

and make this even *more* magical

Reply:
`staging` would need the process.env values, too.  But, yeah, magic seems okay in this case.